example-visitors/src/main/kotlin/kotlinx/serialization/schema/JsonSchema.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 JetBrains s.r.o.
```

Comment:
The year is expired.
You can use `2018-$today.year` macro in copyrights

Reply:
I have it in macro, but have forgotten to update it after copying files